x/group/keeper/keeper_test.go

```diff
@@ -2534,7 +2548,7 @@ func (s *TestSuite) TestProposalsByVPEnd() {
 			admin:     proposers[0],
 			newCtx:    ctx.WithBlockTime(now.Add(votingPeriod).Add(time.Hour)),
 			tallyRes:  group.DefaultTallyResult(),
-			expStatus: group.PROPOSAL_STATUS_SUBMITTED,
+			expStatus: group.PROPOSAL_STATUS_REJECTED,
```

Comment:
we should also update the test name just above:
"tally updated after voting **period** end"